### ISSUE TITLE
spaceAPI

### ISSUE THREAD

**Issue by elf-pavlik** [apollo-ng/dspace-app-apollong#4]
http://hackerspaces.nl/spaceapi/

**Comment by nilclass** [comment]
pseudocode:

```
result = GET http://openspace.slopjong.de/directory.json
for key in result {
  space = GET result[key]
  if space.lat && space.lon {
    // add it!
  }
}
```

**Comment by elf-pavlik** [comment]
:) we can for now use this directory as default through config, still it should take any array of spaceAPI endpoints or event homepages which allow to discover spaceAPI through link relation

we can play with GeoFeedSpaceAPI for that!

**Comment by slopjong** [comment]
Due to a technical reason I have to move lat and lon to the location field in future space api versions. I noticed yesterday that one hackerspace was using long instead of lon which could not be tracked by the validator.

No space has adapted their json until now. I'd like to wait to make this public on the hackerspaces mailing list until freezing 0.13 which is somewhat in next two/three weeks.

**Comment by elf-pavlik** [comment]
@slopjong thanks for giving us a heads up!

**Comment by elf-pavlik** [comment]
@slopjong one more thing... we plan next hackathon on DSpace in Munich for 1-3 March, feel invited to join us :)
http://muc.ccc.de/

**Comment by yggi** [comment]
merged #69
